packages/dx-grid-core/src/plugins/table-header-row/helpers.ts

```diff
@@ -57,3 +59,21 @@ export const generateSimpleChains: GenerateChainsFn = (rows, columns) => (
     start: 0,
   }]))
 );
+
+export const getLastColumnName: LastColumnNameFn = (tableColumns) => {
+  const index = tableColumns.length - 1;
+  return index >= 0 && tableColumns[index].type === TABLE_DATA_TYPE
+    ? tableColumns[index].column!.name
+    : undefined;
+};
+
+export const getNextColumnName: NextColumnNameFn = (tableColumns, columnName) => {
+  const index = tableColumns
+    ? tableColumns.findIndex(elem =>
+      elem.type === TABLE_DATA_TYPE && elem.column!.name === columnName,
```

Comment:
Why are you using a ternary operator here? According to type the `tableColumns` should not be undefined.